lib/mqtt/CMakeLists.txt

```diff
@@ -1,28 +1,40 @@
 afr_module(mqtt)
 
+# Compile BLE MQTT serializers on supported platforms.
+if(${BLE_SUPPORTED})
+    set(extra_mqtt_sources "${AFR_MODULES_DIR}/mqtt/iot_ble_mqtt_serialize.c")
+    set(extra_mqtt_private_includes ${BLE_LIBRARY_INCLUDE_DIR})
+    set(extra_mqtt_dependencies AFR::serializer)
```

Comment:
Should we add `AFR:ble` to the dependencies list here? Instead of adding `BLE_LIBRARY_INCLUDE_DIR` to `extra_mqtt_private_includes`